src/components/Notification/Notification.tsx

```diff
@@ -0,0 +1,201 @@
+import React, { useState } from "react";
+
+import bem from "../../utils/bem";
+import Button from "../Button/Button";
+import { ButtonTypes } from "../Button/ButtonTypes";
+import Heading from "../Heading/Heading";
+import Icon from "../Icons/Icon";
+import { IconColors, IconNames } from "../Icons/IconTypes";
+import { NotificationTypes } from "./NotificationTypes";
+
+export interface NotificationProps {
+  /** BlockName for use with BEM. See how to work with blockNames and BEM here: http://getbem.com/introduction/ */
+  blockName?: string;
+  /** Optional prop to control horizontal alignment of the `Notification` content */
+  centered?: boolean;
+  /** Optional `className` that appears in addition to "notification" */
+  className?: string;
+  /** Optional prop to control whether a `Notification` can be dismissed (closed) by a user */
+  dismissible?: boolean;
+  /** Optional custom `Icon` that will override the default `Icon` */
+  icon?: React.ReactNode;
+  /** ID that other components can cross reference for accessibility purposes */
+  id?: string;
+  /** Optional prop to control the margin around the `Notification` component */
+  noMargin?: boolean;
+  /** Optional prop to control the coloring of notification text and the visibility of an applicable icon */
+  notificationType?: NotificationTypes;
+  /** Modifiers array for use with BEM. See how to work with modifiers and BEM here: http://getbem.com/introduction/ */
+  modifiers?: string[];
+}
+
+// NotificationHeading child-component
+export function NotificationHeading({ ...props }) {
```

Comment:
`NotificationHeading` and `NotificationContent` also have their own file declarations so these should be cleaned up. Both declarations are different from the file components.

Also, it's not necessary to do `NotificationHeading({ ...props })` and `NotificationHeading(props)` is the same.

Reply:
Ah, an oversight.  I started with the separate files, but eventually moved everything within the main `Notification.tsx` file.  Do you have a preference (internal definitions or separate files)?  Pros and cons of the two methods?